pm_muairss_write/pm_muairss_write.xml

```diff
@@ -3,27 +3,43 @@
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
         pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
-        cp $structure input.cell &&
-        cp $params params.yaml &&
+        ln -s $structure input.cell &&
+        ln -s $params params.yaml &&
+        #if str($iscastep.iscastep_sel)=="yes":
+            ln -s $castep_param castep.param &&
+        #end if
         pm-muairss -t w input.cell params.yaml &&
-        zip -r out_zip.zip $out_folder &&
-        find $out_folder > tree.txt
+        out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
+        #if str($iscastep.iscastep_sel)=="yes":
+            zip -r out_zip1.zip \$out_folder params.yaml input.cell castep.param &&
+        #else if str($iscastep.iscastep_sel)=="no":
+            zip -r out_zip1.zip \$out_folder params.yaml input.cell &&
+        #end if
+        ln -s out_zip1.zip out_zip.zip &&
```

Comment:
I was trying to set up the output as a symlink, but there doesnt seem to be much difference in execution time, so i might as well keep the output as the original file

Reply:
I don't think there is any point doing the symlink here – for inputs it is beneficial because the files already exist in the Galaxy history and we don't have to use up extra space by copying them, but for outputs we have created something new so I don't think the symlink matters.

Reply:
Oops, you had already done this and I was looking at the wrong commit. My bad!